File: api/about.js
module.exports = (req, res) => {
  // Set CORS headers
  res.setHeader('Access-Control-Allow-Origin', '*');
  res.setHeader('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, OPTIONS');
  res.setHeader('Access-Control-Allow-Headers', 'Content-Type, Authorization');
  
  if (req.method === 'OPTIONS') {
    res.status(200).end();
    return;
  }
  
  const aboutData = {
    name: "Zahid Rashid",
    title: "AI Engineer & Machine Learning Specialist", 
    description: "Passionate AI Engineer with expertise in machine learning, deep learning, and data science. I specialize in developing innovative AI solutions, building predictive models, and integrating cutting-edge AI technologies into business applications to drive digital transformation.",
    email: "zahid@example.com",
    phone: "+1234567890",
    location: "Your Location",
    profile_image: "/zahid-profile.jpeg",
    years_experience: 3,
    ai_projects: 25,
    ml_models: 50,
    accuracy_rate: 98
  };
  
  if (req.method === 'GET') {
    res.status(200).json(aboutData);
  } else if (req.method === 'PUT') {
    // For demo purposes, just return the updated data
    const updates = req.body || {};
    const updatedData = { ...aboutData, ...updates };
    res.status(200).json(updatedData);
  } else {
    res.status(405).json({ error: 'Method not allowed' });
  }
};

File: api/about.py
from http.server import BaseHTTPRequestHandler
import json
import sys
import os
import sqlite3

# Add the backend directory to the Python path
sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'backend'))

class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        try:
            # Database path for Vercel
            db_path = '/tmp/portfolio.db'
            
            # Sample about info
            about_info = {
                "id": 1,
                "title": "AI Engineer & Machine Learning Specialist",
                "description": "Passionate AI Engineer with expertise in machine learning, deep learning, and data science. I specialize in developing innovative AI solutions, building predictive models, and integrating cutting-edge AI technologies into business applications to drive digital transformation.",
                "profile_image": "/assets/zahid-profile.jpeg",
                "resume_url": "/resume.pdf",
                "created_at": "2024-01-01T00:00:00Z",
                "updated_at": "2024-01-01T00:00:00Z"
            }
            
            self.send_success_response(about_info)
                
        except Exception as e:
            self.send_error_response(500, f"Server error: {str(e)}")
    
    def do_OPTIONS(self):
        self.send_response(200)
        self.send_header('Access-Control-Allow-Origin', '*')
        self.send_header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
        self.send_header('Access-Control-Allow-Headers', 'Content-Type, Authorization')
        self.end_headers()
    
    def send_success_response(self, data):
        self.send_response(200)
        self.send_header('Content-type', 'application/json')
        self.send_header('Access-Control-Allow-Origin', '*')
        self.send_header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
        self.send_header('Access-Control-Allow-Headers', 'Content-Type, Authorization')
        self.end_headers()
        self.wfile.write(json.dumps(data).encode())
    
    def send_error_response(self, code, message):
        self.send_response(code)
        self.send_header('Content-type', 'application/json')
        self.send_header('Access-Control-Allow-Origin', '*')
        self.send_header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
        self.send_header('Access-Control-Allow-Headers', 'Content-Type, Authorization')
        self.end_headers()
        self.wfile.write(json.dumps({"detail": message}).encode())